1_network.yml

```diff
@@ -31,6 +32,33 @@ targets:
     command: gd_get('1_network/in/delaware_stream_temp_by_segment.zip.ind')
   1_network/in/delaware_stream_temp_by_segment/delaware_segments/delaware_segments.shp:
     command: unzip('1_network/in/delaware_stream_temp_by_segment.zip', exdir=I('1_network/in'))
+    
+  #download reservoirs and dams from drive
+  #gd_confirm targets are only used to intialize indicator files, should be left commented
+  #1_network/in/dams-rev01-global-shp.zip.ind:
+  #  command: gd_confirm_posted(target_name)
+  1_network/in/dams-rev01-global-shp.zip:
+    command: gd_get('1_network/in/dams-rev01-global-shp.zip.ind')
+  1_network/out/global_dams.rds.ind:
+    command: zipped_shp_to_rds(out_ind = target_name,
+      zip = '1_network/in/dams-rev01-global-shp.zip', 
+      extract_dir = I('1_network/in/dams-rev01-global-shp'))
```

Comment:
cannot resist urge to comment on PR.....halp

Is the `extract_dir` needed here? If you are not planning on referencing the unzipped shape file files in any way in the future, do you want to throw them away instead? Otherwise, they seem to take up unnecessary local space. What if you unzip to a temp dir (and that way you don't need the additional argument) and/or unzip to temp and then `unlink` after reading as `sf`?

Reply:
I didn't want to unzip directly into `tmpdir`, since then it would be clunky to delete the contents without blowing away everything else in `tmpdir` that is created for the R session.  But I could just create these same folders in `tmpdir` using the exdir argument.  I suppose it doesn't need to be exposed to the user.

Reply:
yeah, you can also create a folder within the tempdir for the unzips to go into and then unlink that whole file, like
```r
unz_dir <- file.path(tempdir(),'shapefile')
dir.create(unz_dir)
# do your thing
unlink(unz_dir)
```